uber/site_sections/registration.py

```diff
@@ -82,6 +85,18 @@ def index(self, session, message='', page='0', search_text='', uploaded_id='', o
     def form(self, session, message='', return_to='', omit_badge='', **params):
         attendee = session.attendee(params, checkgroups=['interests','requested_depts','assigned_depts'],
                                     bools=['staffing','trusted','international','placeholder','got_merch','can_spam'])
+
+        if 'watchlist_id' in params:
+            watchlist_entry = session.watch_list(params['watchlist_id'])
+            watchlist_entry.disabled = True
+            session.add(watchlist_entry)
```

Comment:
You don't need to add the entry to the session here, since any time you query an existing thing from the database, it's always automatically attached already.